torch/csrc/utils/python_arg_parser.cpp

```diff
@@ -184,24 +184,11 @@ auto combine_self_args(PyObject *self, PyObject *args) -> py::tuple {
 }
 
 auto handle_torch_function(PyObject* self, const std::string& func_name, PyObject* args, PyObject* kwargs, PyObject* torch_api, const std::string& module_name) -> PyObject* {
+  std::vector<py::handle> overloaded_args{self};
```

Comment:
NIT: Consider changing `handle_torch_function_no_python_arg_parser` to take `ArrayRef` for `overloaded_args` so we don't need the extra allocation here.